tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt

```diff
@@ -0,0 +1,115 @@
+test-data "conditional choice"
+	category "savegame"
+	contents
+		pilot Bobbi Bughunter
+		date 16 11 3013
+		system "Terra Incognita"
+		planet Ruin
+		clearance
+		ship "Star Barge"
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 10
+				bunks 3
+				"cargo space" 50
+				drag 1
+				"engine capacity" 400
+				"fuel capacity" 300
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 1300
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 200
+			outfits
+				"X1700 Ion Thruster"
+				"X1200 Ion Steering"
+				Hyperdrive
+				"nGVF-BB Fuel Cell"
+				"LP036a Battery Pack"
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			position 0 0
+			engine -9 38 1
+			engine 9 38 1
+			turret 0 -8
+			leak leak 60 50
+			explode "tiny explosion" 10
+			explode "small explosion" 10
+			system "Terra Incognita"
+			planet Ruin
+		account
+			credits 100000000
+			score 400
+			history
+		visited "Terra Incognita"
+		"visited planet" Ruin
+		conditions
+			"%TEST%: CONDITIONAL CHOICE"
+			# to skip the landing on ruin conversation
+			"Ruin: Landing: offered"
+
+
+mission "%TEST%: CONDITIONAL CHOICE"
+	priority
+	source "Ruin"
+	to offer
+		has "%TEST%: CONDITIONAL CHOICE"
+	on offer
+		conversation
+			# cant actually write an integration test for seeing this or not
+			`You should not see this.`
+				to display
+					not "%TEST%: CONDITIONAL CHOICE"
+			`Does this work? Do you not see the first sentence?`
+			choice
+				`	"No."`
+					to display
+						has "%TEST%: CONDITIONAL CHOICE"
+					accept
+				`	"Yes."`
+					decline
+	on accept
+		set "succeed test"
+
+test "Land On Ruin helper"
```

Comment:
This helper function is already in master, no need to repeat it.

Reply:
https://github.com/endless-sky/endless-sky/pull/7623 didnt get merged yet?

Reply:
When it will be it should be removed for sure though